uvicorn/middleware/debug.py

```diff
@@ -81,6 +96,7 @@ async def inner_send(message):
                 raise exc from None
 
             accept = get_accept_header(scope)
+            response: Union[HTMLResponse, PlainTextResponse]
```

Comment:
It can be either this or `response: Response`. Any preference?

Reply:
lgtm that way